controllers/v1/solutionsController.js

```diff
@@ -516,6 +516,32 @@ module.exports = class Solutions extends Abstract {
     });
   }
 
+   /**
+   * Update solution.
+   * @method
+   * @name updateSolutions
+   * @param {Object} req - requested data.
+   * @param {String} req.query.solutionExternalId -  solution external id.
+   * @returns {JSON}
+   */
+
+   async updateSolutions(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let solutionData = await solutionsHelper.updateSolutions(req)
```

Comment:
@praveenKDass we dont need to pass entire req object, whatever relevent that u can pass like body query etc